requirements.txt

```diff
@@ -1,9 +1,9 @@
-torchvision==0.10.0
+torchvision<=0.12.0
 jsonargparse[signatures]
 wandb==0.12.16
 gradio==2.9.4
 pyyaml==5.4.0
 protobuf<4.21.0  # 4.21 breaks with wandb, tensorboard, or pytorch-lightning: https://github.com/protocolbuffers/protobuf/issues/10048
 lightning_hpo
 websockets
-lightning
\ No newline at end of file
+lightning
```

Comment:
Where is the difference here :) ?

Reply:
line endings and this repo does not use auto-formatter